tests/typecheck/test_request.yml

```diff
@@ -106,3 +106,15 @@
         reveal_type(req.GET)  # N: Revealed type is "django.http.request._ImmutableQueryDict"
         req.GET['foo'] = 'bar'  # E: This QueryDict is immutable.  [misc]
         x = 1  # E: Statement is unreachable  [unreachable]
+
+-   case: request_user_auser
```

Comment:
Thanks for it, but I don't think we need this test. The change is simple enough.

Reply:
Yeah, i see my test broke the pipeline. I removed test